### PR TITLE
MY EYES! - In which I revisit blinds again.

### PR DESCRIPTION
**Pull request by OneOneThreeEight**
Complete. Didn't actually want to buff flashbangs and flashpowder is actually so insanely powerful already that I refuse to buff it. Same deal with vampires, but I did so anyway

Essentially, buffs up handheld flashes and other flash sources to render people confused. Fixes an issue where mounted flashes and portable flashes didn't flick with "e_flash". Also adds confusion and dizzying effects if you happen to get blinded by any of the above. *The core part of the change is that some common forms of blinds will now dizzy and confuse.*

Primarily intended to be most useful for bad guy synthetics and malfunctioning AIs defending their core. Notably, flash rounds have also been buffed to have similar dazing effects, but are useless if you don't blind the person.

Being confused now has a 1/4th chance of making you step to a random tile instead of 1/5th.

### CODE REVIEW

**Comment by OneOneThreeEight** [comment]
!wip

**Comment by OneOneThreeEight** [comment]
!review

File: code/game/gamemodes/cult/runes/stun.dm [hunk 1]
```diff
@@ -20,6 +20,8 @@
 				C.stuttering = 1
 			if(is_rune)
 				C.Weaken(3)
+			C.make_dizzy(60)
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/cult/runes/stun.dm]: This does not actually do anything. It has to be over 100 for dizziness to process. Imho the stun runes don't need dizzy.

File: code/game/gamemodes/cult/runes/stun.dm [hunk 2]
```diff
@@ -20,6 +20,8 @@
...
+			C.confused = rand(5,15)
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/cult/runes/stun.dm]: This really should not be random with such a large interval.

File: code/game/objects/items/devices/flash.dm [hunk 3]
```diff
@@ -79,6 +79,8 @@
 		var/safety = M:eyecheck(TRUE)
 		if(safety <= 0)
 			flick("e_flash", M.flash)
+			M.make_dizzy(60)
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/flash.dm]: This doesn't do anything and flashes shouldn't have it.

**Comment by OneOneThreeEight** [comment]
Didn't know dizzy worked like that -- or rather didn't work at all. Changed either way.

Removed the randomness on request, might be better if it's more predictable actually.

File: code/modules/projectiles/projectile/energy.dm [hunk 4]
```diff
@@ -23,6 +23,8 @@
 	//blind adjacent people
 	for (var/mob/living/carbon/M in viewers(T, flash_range))
 		if(M.eyecheck() < FLASH_PROTECTION_MODERATE)
+			M.make_dizzy(60)
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/projectile/energy.dm]: This doesn't do anything.

**Pull request merged**